src/types/timezone.jl

```diff
@@ -1,5 +1,6 @@
 # Retains the compiled tzdata in memory. Read-only access is thread-safe and any changes
 # to this structure can result in inconsistent behaviour.
+# Do not access this object directly, instead use `get_tz_cache()` to access the cache.
 const _TZ_CACHE = Dict{String,Tuple{TimeZone,Class}}()
```

Comment:
Let's keep the constants together
```suggestion
const _TZ_CACHE = Dict{String,Tuple{TimeZone,Class}}()
const _TZ_CACHE_INITIALIZED = Threads.Atomic{Bool}(false)
const _TZ_CACHE_LOCK = ReentrantLock()
```